jiant/tasks/edge_probing.py

```diff
@@ -178,19 +178,10 @@ def get_num_examples(cls, split_text):
     def _make_span_field(cls, s, text_field, offset=1):
         return SpanField(s[0] + offset, s[1] - 1 + offset, text_field)
 
-    def _pad_tokens(self, tokens):
-        """Pad tokens according to the current tokenization style."""
-        if self.tokenizer_name.startswith("bert-"):
-            # standard padding for BERT; see
-            # https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/examples/extract_features.py#L85  # noqa
-            return ["[CLS]"] + tokens + ["[SEP]"]
-        else:
-            return [utils.SOS_TOK] + tokens + [utils.EOS_TOK]
-
-    def make_instance(self, record, idx, indexers) -> Type[Instance]:
+    def make_instance(self, record, idx, indexers, boundary_token_fn) -> Type[Instance]:
         """Convert a single record to an AllenNLP Instance."""
         tokens = record["text"].split()  # already space-tokenized by Moses
-        tokens = self._pad_tokens(tokens)
+        tokens = boundary_token_fn(tokens)
```

Comment:
nit: add comment on what this does, since it's not obvious from the immediate context anymore?
```
tokens = boundary_token_fn(tokens)  # add model-appropriate padding (e.g. [CLS] and [SEP] for BERT)
```